### PR TITLE
Final cleanup of AppSync tests

### PR DESCRIPTION
**Pull request by rjuliano**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by rjuliano** [comment]
> I believe the `android` command is deprecated, now. Can you avoid it?

Trying that right now. Needs a couple more components installed in order to do that. The first time I tried, it couldn't find the avdmanager command.

File: README.md [hunk 1]
```diff
@@ -402,6 +402,35 @@ cd aws-android-sdk-appsync-tests/
 
 To run tests from Android Studio, run `./gradlew publishToMavenLocal` from project root, then open the `aws-android-sdk-appsync-tests/` directory in Android Studio, and run the tests through the UI.
 
+### Integration testing infrastructure
```
Comment (jamesonwilliams) [inline README.md:405]: Since this is an unlikely branch for the average reader, should we put it into a `CONTRIBUTING.md`? Or, maybe a `TESTING.md`?

Comment (rjuliano) [reply README.md:405]: Ah...that's a good point. It felt kind of out of place. I'll move it to an separate file.

File: .travis.yml [hunk 2]
```diff
@@ -3,32 +3,27 @@ before_install:
...
 - wc aws-android-sdk-appsync-tests/src/main/res/raw/awsconfiguration.json
 - chmod +x gradlew
-- yes | sdkmanager "platforms;android-25"
-- yes | sdkmanager tools
-- sdkmanager "system-images;android-25;google_apis;armeabi-v7a"
+- yes | sdkmanager "platforms;android-27"
 - android list target
-- echo no | avdmanager create avd --force -n test -k "system-images;android-25;google_apis;armeabi-v7a"
-- $ANDROID_HOME/emulator/emulator -avd test -no-skin -no-audio -no-window &
+- echo no | android create avd --force -n test -t android-22 --abi armeabi-v7a
```
Comment (jamesonwilliams) [inline .travis.yml]: Don't you have to change all of the values to the same thing? Line 6 says `android-27`, but this says `android-22`?

Comment (rjuliano) [reply .travis.yml]: Yeah...and the weird thing is that it's what it is currently in master. I'm just reverting the changes I made to this file for this PR and will pick that up separately.

**Pull request merged**